docs/source/user_guide/pipelines.md

```diff
@@ -314,16 +342,22 @@ To run a pipeline from the Visual Pipeline Editor:
 
    ![Open pipeline run wizard](../images/user_guide/pipelines/pipeline-editor-run.png)
 
-1. For generic pipelines select a runtime platform (local, Kubeflow Pipelines, Apache Airflow) and a runtime configuration for that platform. For runtime-specific  pipelines select a runtime configuration.
+2. For generic pipelines select a runtime platform (local, Kubeflow Pipelines, Apache Airflow) and a runtime configuration for that platform. For runtime-specific  pipelines select a runtime configuration.
 
    ![Configure pipeline run options](../images/user_guide/pipelines/configure-pipeline-run-options.png)
 
-1. Elyra does not include a pipeline run monitoring interface for pipelines:
+3. Configure pipeline parameters, if applicable. If any nodes reference parameters defined in the `Pipeline Parameters` panel, the value these parameters take can be customized here. If a parameter is marked as required and no default value is set, a value must be provided before the `OK` button is enabled.
+
+   ![Configure pipeline export options with parameters](../images/user_guide/pipelines/configure-pipeline-submit-options-parameters.gif)
```

Comment:
ALT description and image name disagree (export vs submit)

Reply:
The image depicts the export dialog, not the submit dialog

Reply:
thank you! I'll re-record both anyway due to the variable name issue above